docs/cedar.md

```diff
@@ -32,13 +32,13 @@ The webMethods API Portal is where you can view CEDARs API documentation, and is
 
 ### Alfabet
 
-Alfabet is the backend API that stores the data that CEDAR pulls from. EASi does not integrate directly with Alfabet, but uses CEDAR as a proxy to Alfabet.
+Alfabet is the backend API that stores the data that CEDAR pulls from. MINT does not integrate directly with Alfabet, but uses CEDAR as a proxy to Alfabet.
 
 However, you can click Alfabet on the main CEDAR UI page to see the Alfabet UI, which lets you browse the data in Alfabet. Do _NOT_ modify any data directly in Alfabet.
 
-## How is CEDAR accessed within EASi?
+## How is CEDAR accessed within MINT?
 
-The calls that are made to the CEDAR APIs are defined by code generated by `go-swagger`. However, in order to not have changes to the CEDAR API documentation break EASi, a "translated client" is usually created for each CEDAR API that wraps the generated code with our own, tested client. An example of this is the [pkg/cedar/intake/client.go](../pkg/cedar/intake/client.go) file.
+The calls that are made to the CEDAR APIs are defined by code generated by `go-swagger`. However, in order to not have changes to the CEDAR API documentation break MINT, a "translated client" is usually created for each CEDAR API that wraps the generated code with our own, tested client. An example of this is the [pkg/cedar/intake/client.go](../pkg/cedar/intake/client.go) file.
```

Comment:
This can get updated too - We don't really use CEDAR for anything in MINT outside the LDAP endpoints. At least the references to the `intake` client can go.